roles/ceph-osd/tasks/docker/start_docker_osd.yml

```diff
@@ -67,6 +67,11 @@
     - ceph_osd_docker_prepare_env is defined
     - osd_containerized_deployment_with_kv
 
+- name: get ceph journal partuuid
+  command: blkid -t PARTLABEL="ceph journal" -o value -s PARTUUID
```

Comment:
Comment to myself, this will grab ALL the journal partition, not the one from the container, I need to be more precise here.